src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -255,6 +256,7 @@ The .NET Foundation licenses this file to you under the MIT license.
 
       <!-- The managed debugging support in libraries is unused - trim it -->
       <IlcArg Condition="'$(IlcKeepManagedDebuggerSupport)' != 'true'" Include="--feature:System.Diagnostics.Debugger.IsSupported=false" />
+      <IlcArg Condition="'$(UseWindowsThreadPool)' != '' and '$(_targetOS)' == 'win' and '$(IlcKeepManagedDebuggerSupport)' == 'true'" Include="--feature:System.Diagnostics.Debugger.IsSupported=$(UseWindowsThreadPool)" />
```

Comment:
What has threadpool to do with IlcKeepManagedDebuggerSupport? This condition does not look right.

Reply:
The CI was failing with some error along the lines of `An item with the same key has already been added. Key: System.Diagnostics.Debugger.IsSupported`. My guess is that both `"'$(IlcKeepManagedDebuggerSupport)' != 'true'"` and `"'$(UseWindowsThreadPool)' != '' and '$(_targetOS)' == 'win'"` evaluated to true and the build couldn't handle a reassignment of the key. I added the extra condition to make them mutually exclusive

Reply:
The actual bug is that the `--feature` name on this line is wrong (copy&paste bug). It needs to be the threadpool feature name.

Reply:
Ah I see, thanks!